jasmine/index.html

```diff
@@ -0,0 +1,305 @@
+&lt;script type="text/javascript" src="src/App.js"&gt;&lt;/script&gt;
+&lt;script type="text/javascript" src="src/Game.js"&gt;&lt;/script&gt;
+
+&lt;!-- include spec files here... --&gt;
+&lt;script type="text/javascript" src="spec/GameSpec.js"&gt;&lt;/script&gt;</code></pre>
+				</section>
+				<section>
+					<h3>Suites, specs, and matchers</h3>
+<pre><code>// Suites start with the keyword "describe"
+describe("A suite", function(){
+
+  // Specs start with the keyword "it"
+  it("contains spec with an expectation", function(){
+
+    // expect() statements have matchers like toBe()
+    expect(true).toBe(true);
+
+  });
+});
+</code></pre>
+				</section>
+				<section>
+					<h3><code>toBe()</code> or <code>not.toBe()</code></h3>
+					<p>There are special functions called <strong>matchers</strong> that will compare an actual value with an expected value.</p>
+					<br>
+					<p>You can also add a <code>.not()</code> to your matcher:</p>
+					<pre><code>expect(true).not.toBe(false);</code></pre>
+					<br>
+					<p>Here are some <a href="http://jasmine.github.io/2.0/introduction.html#section-Matchers" target="_blank">other matchers</a> that are built-in:</p>
+<pre><code>.toBeDefined();
+.toEqual(); // strict equality
+.toMatch(); // for regular expressions
+.toContain(); // find an item in an array
+</code></pre>
+				</section>
+				<section>
+					<h3>A pattern for executing tests</h3>
+					<br>
+					<ul>
+						<li>Set up the stage</li>
+						<li>Let the players play their parts</li>
+						<li>Tear down the stage</li>
+					</ul>
+				</section>
+				<section>
+					<h3>Set up and Tear down</h3>
+					<p>Often, you'll run tests with the same set up and tear down procedures. For that, we can use the <code>beforeEach()</code> and <code>afterEach()</code> functions:</p>
+					<pre><code>describe("a self-driving car", function(){
+  describe("turning in traffic", function(){
+    beforeEach(function(){
+      startEngine();
+      navigate("Safeway");
+    });
+
+    afterEach(function(){
+      navigate("Home");
+      killEngine();
+    });
+
+    it("should signal when turning", function(){
+      turnLeft();
+      expect(leftSignalPosition).toEqual("on");
+    });
+  });
+});
+					</code></pre>
+					<p><a href="http://jasmine.github.io/2.0/introduction.html#section-Setup_and_Teardown" target="_blank">more from the docs</a></p>
+				</section>
+				<section>
+					<h3>Exercise time</h3>
+					<br>
+					<p><a href="step1/instructions.html" target="_blank">Click here for directions</a></p>
+					<p>Suggested time: 30 minutes</p>
+				</section>
+
+				<!-- Jasmine Part 2 -->
+				<section>
+					<h3>Control Flow</h3>
+					<h4>Shoulda, coulda, woulda</h4>
+					<p><strong>Control Flow</strong> is the order in which the individual statements, instructions or function calls are executed or evaluated.</p>
+					<br>
+					<p>Because this can get complicated quickly, we'll need to test this.</p>
```

Comment:
Hm. Not sure I understand the distinction between testing control flow and testing other aspects, can you clarify? This slide feels a bit ambiguous to me. Maybe an example?